editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -773,6 +786,14 @@ void Polygon2DEditor::_uv_input(const Ref<InputEvent> &p_input) {
 						undo_redo->add_do_method(uv_edit_draw, "queue_redraw");
 						undo_redo->add_undo_method(uv_edit_draw, "queue_redraw");
 						undo_redo->commit_action();
+					} else if (uv_move_current == UV_MODE_MOVE_PIVOT) {
+						undo_redo->create_action(TTR("Set offset"));
```

Comment:
```suggestion
						undo_redo->create_action(TTR("Set Offset"));
```
Undo-redo style

Reply:
Hmm, if I change offset from inspector it prints "Set offset", I matched to that. Or is the style different when working with editors vs inspector, which seems to be printing raw property name?

Reply:
That's different, see the other cases above, title case is used here

Reply:
Okay, nice to know, thanks! I'll change it 👌